hack/image/push-image-tags.sh

```diff
@@ -175,4 +214,8 @@ function is_latest_tag() {
   [[ -z "$latest" || "$candidate" == "$latest" ]]
 }
 
-push_image_tags "$@"
+if [ "$#" == "2" ]; then
+    push_image_tags "$@"
+else
+    push_manifest_list "$@"
+fi
```

Comment:
Could we split the `push_manifest_list` function out into a separate file, and move any shared functions to another lib file (e.g. `hack/lib/image-helpers.sh`)

Reply:
ah, sure